.typo-ci.yml

```diff
@@ -1,6 +1,7 @@
 dictionaries:
   - en
   - en_GB
+  - pt

```

Comment:
I set this to be `pt`, I'm not that familiar with the Portuguese language but if you want you can also set this to be `pt_BR`.

Reply:
great job Mike  👍 👍

Reply:
Thanks you @donnisnoni95 :)